portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+/*A comment from the comments section in the home page*/
+public final class Comment {
+
+  private final Key id;
+  private final String title;
+  private final String dateAndTime;
```

Comment:
Should "date" be sufficient?

Reply:
umm okay, but I thought it should be more expressive as it also shows time.